### DISCUSSION TITLE
How does one fix, Echo: Unknown Command: "*"

### DISCUSSION THREAD

**Discussion by Valerie-Kyrie** [luc-github/ESP3D#618] [Questions]
I am running a SKR 1.4 turbo running Marlin (ver: 020008), TFT50 (ver: 20210321) with ESP-01s (ver: 2.1.1.b8) attached. The setup seems to mostly work. I can see temperatures, move the steppers, and level my bed. Basically the main functionality seems to be working.
However, when I go to the page "ESP3D for Marlin" Printer Configuration and push the 'refresh' button on the page, the error
Echo:
Unknown Command: "*"
appears on my TFT50. How would I correct this?

**Comment by luc-github** [comment]
what is the version of webui?

**Reply by Valerie-Kyrie** [reply]
Sorry I missed that part of the info for you, UI: 2.1b70 / FW: 2.1.1.b8

**Comment by luc-github** [comment]
are you sure it is webUI 2.1b70 ? the M503 sending * has been fixed 2 years ago

now no command send any `*`  since 2.1b24

**Reply by Valerie-Kyrie** [reply]
Yes I am quite sure, as that UI and FW number was copied and pasted from the web UI, when I hovered the mouse over the menu icon in the upper right corner.

**Reply by luc-github** [reply]
can you open your browser devtool ?

**Reply by Valerie-Kyrie** [reply]
I am unsure why, but after opening my browsers devtools and pressing the refresh on the printer configuration page, the error is no longer appearing, replaced with 
Echo: 
Z2.00
which I believe? is correct, as this is not an unknown command. 
Thank you, for helping me sort this out.

**Reply by luc-github** [reply]
this is weird  - not sure what happened
